src/chainlit/langchain/callbacks.py

```diff
@@ -286,6 +293,9 @@ def _on_llm_start(
     invocation_params = kwargs.get("invocation_params")
     provider, settings = get_llm_settings(invocation_params)
 
+    # TODO: path not tested yet
+    provider = ".".join(serialized.get("id", []))
```

Comment:
provider is defined just above with get_llm_settings

Reply:
yes, I had to redefine the provider for the purpose of automation. I still need to clean up the `get_llm_settings` function.

Right now the `get_llm_settings` returns something like `openai-chat`, while we are now using the full langchain class path as the provider id (extracted from the `serialized` dict)
This enables us to access the llm id from places where the `invocation_params` aren't configured.

Reply:
we've moved back to the langchain id, like `openai-chat` instead of the full class path